src/Shared/HttpSys/Extensions.cs

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.AspNetCore.HttpSys.Internal
+{
+    internal static class Extensions
+    {
+        public static string GetHttpProtocolVersion(this Version version) => version switch
```

Comment:
Ooooh, fancy new C# 8.0 features 😜

Reply:
:dart: it's a better showcase than [the one from the docs](https://docs.microsoft.com/en-us/dotnet/csharp/whats-new/csharp-8#property-patterns) 😃